src/components/CompareResults/loader.ts

```diff
@@ -180,22 +180,32 @@ export async function loader({ request }: { request: Request }) {
       generation: generationCounter++,
     };
   }
-
-  let baseRevFromUrl = url.searchParams.get('baseRev');
-  let newRevsFromUrl = url.searchParams.getAll('newRev');
+  const baseRevFromUrl = url.searchParams.get('baseRev');
+  const newRevsFromUrl = url.searchParams.getAll('newRev');
   const baseHashFromUrl = url.searchParams.get('baseHash');
   const newHashFromUrl = url.searchParams.get('newHash');
+
+  // This loader is the only one used by ./mach try perf, and due to
+  // recent changes mach try perf can't get instant push links to try
+  // when we push, we must attach a hash to the commit message and
+  // search commits for that hash, and return the commit associated
+  // with that hash and update the baseRev and newRev
+  let baseRevsFromHash = baseRevFromUrl;
+  let newRevsFromHash = newRevsFromUrl;
+  let usingRevisionsOrHash = true;
```

Comment:
Nit: Wondering if the naming here could be clearer. Something like  `notUsingHashToFindRevisions` or something of the sort.

Reply:
I've change to use:  `notUsingHashToFindRevisions`